client/components/header-component.js

```diff
@@ -1,14 +1,19 @@
 const HeaderComponent = {
-  template: `<div class="header">
-  <a class="link" href="/">
-    <img alt="HYF_connect logo" src="/public/assets/logo_pc_full.png" width="90"/>
-  </a>
-  <a class="link" href="/pages/projects/projects.html">projects</a>
-  <a class="link" href="/pages/members/members.html">members</a>
-  <a class="link" href="/pages/about-us/about-us.html">about us</a>
-  <a class="link" href="/pages/contact-us/contact-us.html">contact us</a>
-  <button>Log In</button>
-</div>`
+    template: `<div class="header-container">
+    <div class="brand">
+            <a href="https:" class="logo"><img src="/public/assets/logo_pc_full.png" alt="logo" width="90" height="70"/></a>
```

Comment:
Clicking on the logo should bring us back to the homepage..
Replace the link of the logo (currently https: ??? with /pages/homepage/homepage.html or just /